### DISCUSSION TITLE
LSP4E 0.26.7 released

### DISCUSSION THREAD

**Discussion by rubenporras** [eclipse/lsp4e#1157] [Announcements]
## New features
* Allow derived MockLanguageServer for derived servers. by @andrewL-avlq in https://github.com/eclipse-lsp4e/lsp4e/pull/1133

## Fixes
* Set the vendor of all plugins to Eclipse LSP4E by @FlorianKroiss in https://github.com/eclipse-lsp4e/lsp4e/pull/1143
* Use merge instead of replacing style from semantic highlighting by @gregdyke in https://github.com/eclipse-lsp4e/lsp4e/pull/1148 and https://github.com/eclipse-lsp4e/lsp4e/pull/1153
* Fix some links by @mickaelistria in https://github.com/eclipse-lsp4e/lsp4e/pull/1150
* Improve exception handling in InlayHintProvider by @joaodinissf in https://github.com/eclipse-lsp4e/lsp4e/pull/1151
* Fix disappearing menu in outline view by @travkin79 in https://github.com/eclipse-lsp4e/lsp4e/pull/1152
* Move the assertion that makes the test fail by @rubenporras in https://github.com/eclipse-lsp4e/lsp4e/pull/1155
* Mark test HoverTest as ignored until it can be fixed by @rubenporras in https://github.com/eclipse-lsp4e/lsp4e/pull/1156

## Build
* fix(deps): bump com.googlecode.maven-download-plugin:download-maven-plugin from 1.9.0 to 1.10.0 in /org.eclipse.lsp4e by @dependabot in https://github.com/eclipse-lsp4e/lsp4e/pull/1134
* fix(deps): bump com.googlecode.maven-download-plugin:download-maven-plugin from 1.10.0 to 1.11.0 in /org.eclipse.lsp4e by @dependabot in https://github.com/eclipse-lsp4e/lsp4e/pull/1135
* fix(deps): bump com.googlecode.maven-download-plugin:download-maven-plugin from 1.11.0 to 1.11.1 in /org.eclipse.lsp4e by @dependabot in https://github.com/eclipse-lsp4e/lsp4e/pull/1137
* fix(deps): bump com.googlecode.maven-download-plugin:download-maven-plugin from 1.11.1 to 1.11.2 in /org.eclipse.lsp4e by @dependabot in https://github.com/eclipse-lsp4e/lsp4e/pull/1138
* fix(deps): bump com.googlecode.maven-download-plugin:download-maven-plugin from 1.11.2 to 1.11.3 in /org.eclipse.lsp4e by @dependabot in https://github.com/eclipse-lsp4e/lsp4e/pull/1142
* fix(deps): bump tycho-version from 4.0.9 to 4.0.10 in /org.eclipse.lsp4e by @dependabot in https://github.com/eclipse-lsp4e/lsp4e/pull/1141
* fix(deps): bump tycho-version from 4.0.9 to 4.0.10 in /repository by @dependabot in https://github.com/eclipse-lsp4e/lsp4e/pull/1140
* fix(deps): bump tycho-version from 4.0.9 to 4.0.10 in /org.eclipse.lsp4e.test by @dependabot in https://github.com/eclipse-lsp4e/lsp4e/pull/1139
* fix(deps): bump com.googlecode.maven-download-plugin:download-maven-plugin from 1.11.3 to 1.12.1 in /org.eclipse.lsp4e by @dependabot in https://github.com/eclipse-lsp4e/lsp4e/pull/1145
* fix(deps): bump com.googlecode.maven-download-plugin:download-maven-plugin from 1.12.1 to 1.13.0 in /org.eclipse.lsp4e by @dependabot in https://github.com/eclipse-lsp4e/lsp4e/pull/1147
* fix target after removal of SimRel milestone P2 site by @rubenporras in https://github.com/eclipse-lsp4e/lsp4e/pull/1154

## New Contributors
* @gregdyke made their first contribution in https://github.com/eclipse-lsp4e/lsp4e/pull/1148

**Full Changelog**: https://github.com/eclipse-lsp4e/lsp4e/compare/0.26.6...0.26.7

<hr /><em>This discussion was created from the release <a href='https://github.com/eclipse-lsp4e/lsp4e/releases/tag/0.26.7'>LSP4E 0.26.7 released</a>.</em>

**Comment by mickaelistria** [comment]
Thank you!
